empresas-lista.py

```diff
@@ -72,7 +80,7 @@ def main(outfile):
     # se assegura que ele exista e envia um clique
 
     allbtn = driver.find_element_by_name(allbtnid)
-
+        
```

Comment:
Remover espaço em linhas vazias

Reply:
Olá Caian. Estou tentando rodar o código para gerar a lista de empresas. Quando uso o asserttitle "Untitled Page", retorna um AssertionError. No meu caso funcionou com asserttitle = "Empresas Listadas". Porém o script não finaliza, gera erro quando tenta encontrar o allbtnid (allbtn = driver.find_element_by_name(allbtnid)): NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"[name="ctl00$contentPlaceHolderConteudo$BuscaNomeEmpresa1$btnTodas"]"}
  (Session info: chrome=79.0.3945.88)